packages/server/src/fhir/routes.ts

```diff
@@ -335,6 +337,8 @@ function initInternalFhirRouter(): FhirRouter {
   router.add('POST', '/$db-stats', dbStatsHandler);
   router.add('POST', '/$db-schema-diff', dbSchemaDiffHandler);
   router.add('POST', '/$db-invalid-indexes', dbInvalidIndexesHandler);
+  router.add('GET', '/$db-column-statistics', lookupDbColumnStatisticsHandler);
+  router.add('POST', '/$db-column-statistics', updateDbColumnStatisticsHandler);
```

Comment:
This doesn't really comport with how FHIIR models Operations: `$db-column-statistics` has two conflicting OperationDefinitions.  These should be split into two different operation routes, each with their own OperationDefinition